pkg/devfile/adapters/common/command_supervisor.go

```diff
@@ -32,9 +32,12 @@ func newSupervisorInitCommand(command common.DevfileCommand, adapter commandExec
 }
 
 // newSupervisorStopCommand creates a command implementation that stops the specified command via the supervisor
-func newSupervisorStopCommand(command common.DevfileCommand, executor commandExecutor) (command, error) {
-	cmd := []string{SupervisordBinaryPath, SupervisordCtlSubCommand, "stop", "all"}
-	if stop, err := newOverriddenSimpleCommand(command, executor, cmd); err == nil {
+func newSupervisorStopCommand(command common.DevfileCommand, cmd string, executor commandExecutor) (command, error) {
```

Comment:
The parameter naming for this method can be improved. first parameter is "command" and the second one is "cmd", which is just short for "command". Will be better to address this and add a small description about the function parameters in the function description comment above.